### PR TITLE
Remove ToMisoString constraint on text

### PR DESCRIPTION
**Pull request by dmjio**
Will get around having to `pack` everything.
- Re-exported `Data.Monoid` from `Miso.String`
- Removed silly `ToMisoString Value` instance.
- Added `ms` function as a concise `toMisoString`

cc @cocreature, can I get a review here.

### CODE REVIEW

File: ghcjs-src/Miso/String.hs [hunk 1]
```diff
@@ -45,11 +47,12 @@ instance FromJSON MisoString where
 -- | Convenience class for creating `MisoString` from other string-like types
 class ToMisoString str where
   toMisoString :: str -> MisoString
+  ms :: str -> MisoString
```
Comment (cocreature) [inline ghcjs-src/Miso/String.hs]: What’s the difference between `toMisoString` and `ms`? Is `ms` just a shorter name? If so, should it really be a class method that instances can potentially override?

File: exe/Main.hs [hunk 2]
```diff
@@ -30,7 +32,8 @@ data Action
 
 viewModel :: Int -> View Action
 viewModel x = div_ [] [
-   button_ [ onClick AddOne ] [ text "+" ]
- , text (show x)
+   button_ [ onClick AddOne ] [ text "foo" ]
```
Comment (cocreature) [inline exe/Main.hs]: That’s probably not an intentional change.

Comment (dmjio) [inline exe/Main.hs]: Indeed :)

File: ghcjs-src/Miso/String.hs (hunk 1 above)
Comment (dmjio) [inline ghcjs-src/Miso/String.hs]: Ah, great point. Will factor out into standalone `ms :: ToMisoString s => s -> MisoString`

**Comment by dmjio** [comment]
Should be fixed now.

**Pull request merged**